python/README.md

```diff
@@ -1,38 +1,27 @@
 # nGraph python binding
 
-## Installation (If you haven't build nGraph yet.)
+## Installation
 
-Checkout nGraph and python wrapper code and build bdist wheel.
+### Install nGraph (Required)
 
 ```
-git clone https://github.com/NervanaSystems/ngraph.git
-cd ngraph/python
+Follow the [steps](http://ngraph.nervanasys.com/docs/latest/install.html) to build and install ngraph.
 ```
 
-```
-pip install wheel
-```
+Set the environment variables
 
-To build python2 bdist wheel type
 ```
-./build_python2_wheel.sh
-```
-To build python3 bdist wheel type
-```
-./build_python3_wheel.sh
+export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
+export LD_LIBRARY_PATH=$HOME/ngraph_dist/lib/
+export DYLD_LIBRARY_PATH=$HOME/ngraph_dist/lib (Only needed on MacOS)
 ```
 
-The bdist wheel will be placed in ngraph/python/build/dist
-Activate your virtual environment and install the bdist wheel
+Install Wrapper (python binding)
 
 ```
-pip install -U <full path to the bdist wheel>
-```
-
-For example, On MacOS you would run a command like,
-
-```
-pip install -U build/dist/ngraph-0.0.1-cp35-cp35m-linux_x86_64.whl
+cd ngraph/python
+git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
+pip install -U .
```

Comment:
I think `python setup.py install` works better (is more verbose) and faster then `pip install .`